extensions/remember-this/src/notifs.tsx

```diff
@@ -0,0 +1,84 @@
+import { List, Icon, openExtensionPreferences, Action, ActionPanel } from "@raycast/api";
+
```

Comment:
I don't think this command is necessary.
You can add instructions on how to use your extension on the README file.
Do not add things that are not there yet to your list.

Reply:
All fixed!